client/recipient_test.go

```diff
@@ -57,6 +57,7 @@ func TestRecipientsEmail(t *testing.T) {
 
 		assert.NoError(t, err, "failed to update Recipient")
 		// copy dynamic field before asserting equality
+		rcpt.CreatedAt = result.CreatedAt
```

Comment:
This is a hack.. the client tests need to be revisited in general as they're starting to show strain under more frequent usage.